src/wp-includes/js/media/controllers/cropper.js

```diff
@@ -104,6 +145,13 @@ Cropper = wp.media.controller.State.extend({
 		this.frame.toolbar.set( new wp.media.view.Toolbar(toolbarOptions) );
 	},
 
+	/**
+	 * @summary Creates an object with the image attachment and crop properties.
+	 *
+	 * @since 4.2.0
+	 *
+	 * @returns object Ajax post with the custom header crop details.
```

Comment:
`object` is wrong. This should be `{Object}`

Reply:
If the `wp.ajax.post` relies on jQuery it might return a `Promise`. While typing this I've looked for some evidence to verify this:
![cursor_and_wp-util_js_-_wordpress-seo-premium_-____documents_development_vvv_www_wordpress-default_public_html_wp-content_plugins_wordpress-seo-premium_](https://user-images.githubusercontent.com/325040/32054953-803912e4-ba60-11e7-9691-fb789d7ad0ae.png)

Even better it is a `$.promise`. Happiness for everyone.